src/main/java/store/mybookstore/mapper/OrderItemMapper.java

```diff
@@ -0,0 +1,14 @@
+package store.mybookstore.mapper;
+
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.Mappings;
+import store.mybookstore.config.MapperConfig;
+import store.mybookstore.dto.orderitem.OrderItemResponseDto;
+import store.mybookstore.model.OrderItem;
+
+@Mapper(config = MapperConfig.class)
+public interface OrderItemMapper {
+    @Mappings({@Mapping(target = "bookId", source = "book.id")})
+    OrderItemResponseDto toDto(OrderItem orderItem);
+}

```

Comment:
I can't solve the problem. The response what i get in order, always show null value for book id in order item. How can i fix that. I try to solve proplem by adding mapped orderitem response in order response but i think there is more common fix.